pkg/registry/core/rest/storage_core.go

```diff
@@ -180,70 +253,71 @@ func (c GenericLegacyRESTStorageProvider) NewRESTStorage(apiResourceConfigSource
 
 	return apiGroupInfo, nil
 }
-func (c LegacyRESTStorageProvider) NewLegacyRESTStorage(apiResourceConfigSource serverstorage.APIResourceConfigSource, restOptionsGetter generic.RESTOptionsGetter) (kubernetesservice.RangeRegistries, genericapiserver.APIGroupInfo, error) {
-	apiGroupInfo, err := c.GenericLegacyRESTStorageProvider.NewRESTStorage(apiResourceConfigSource, restOptionsGetter)
+
+func (p *GenericConfig) GroupName() string {
+	return api.GroupName
+}
```

Comment:
nit , you used  `(c *GenericConfig) ` before

Reply:
indeed. Fixing.

Reply:
fixed